Python/PRP/GUI/pyGUIControlMod.cpp

```diff
@@ -0,0 +1,179 @@
+    pyGUIColorScheme_transparent_getset,
+    pyGUIColorScheme_fontFace_getset,
+    pyGUIColorScheme_fontSize_getset,
+    pyGUIColorScheme_fontFlags_getset,
+    PY_GETSET_TERMINATOR
+};
+
+PY_PLASMA_TYPE(GUIColorScheme, pfGUIColorScheme, "pfGUIColorScheme wrapper");
+
+PY_PLASMA_TYPE_INIT(GUIColorScheme) {
+    pyGUIColorScheme_Type.tp_new = pyGUIColorScheme_new;
+    pyGUIColorScheme_Type.tp_getset = pyGUIColorScheme_GetSet;
+    if (PyType_Ready(&pyGUIColorScheme_Type) < 0)
+        return NULL;
+
+    /* Konstants */
+    PY_TYPE_ADD_CONST(GUIColorScheme, "kFontBold", pfGUIColorScheme::kFontBold);
+    PY_TYPE_ADD_CONST(GUIColorScheme, "kFontItalic", pfGUIColorScheme::kFontItalic);
+    PY_TYPE_ADD_CONST(GUIColorScheme, "kFontShadowed", pfGUIColorScheme::kFontShadowed);
+
+    Py_INCREF(&pyGUIColorScheme_Type);
+    return (PyObject*)&pyGUIColorScheme_Type;
+}
+
+PY_PLASMA_VALUE_IFC_METHODS(GUIColorScheme, pfGUIColorScheme)
+
+}
+
+extern "C" {
+
+PY_PLASMA_NEW(GUIControlMod, pfGUIControlMod)
+
+PY_METHOD_NOARGS(GUIControlMod, clearSoundIndices, "Remove all sound indices from the control") {
+    self->fThis->clearSoundIndices();
+    Py_RETURN_NONE;
+}
+
+static PyMethodDef pyGUIControlMod_Methods[] = {
+    pyGUIControlMod_clearSoundIndices_method,
+    PY_METHOD_TERMINATOR
+};
+
+PY_GETSET_GETTER_DECL(GUIControlMod, soundIndices) {
+    PyObject* list = PyTuple_New(self->fThis->getSoundIndices().size());
+    for (size_t i = 0; i<self->fThis->getSoundIndices().size(); i++)
+        PyTuple_SET_ITEM(list, i, pyPlasma_convert(self->fThis->getSoundIndices()[i]));
+    return list;
+}
+
+PY_GETSET_SETTER_DECL(GUIControlMod, soundIndices) {
+    PY_PROPERTY_CHECK_NULL(soundIndices)
+        if (value == Py_None) {
+            self->fThis->setSoundIndices(std::vector<int>());
+            return 0;
+        }
+    pySequenceFastRef seq(value);
+    if (!seq.isSequence()) {
+        PyErr_SetString(PyExc_TypeError, "soundIndices should be a sequence of ints");
+        return -1;
+    }
+    std::vector<int> soundIndices(seq.size());
+    for (size_t i = 0; i<soundIndices.size(); i++) {
+        PyObject* itm = seq.get(i);
+        if (!pyPlasma_check<int>(itm)) {
+            PyErr_SetString(PyExc_TypeError, "soundIndices should be a sequence of ints");
+            return -1;
+        }
+        soundIndices[i] = pyPlasma_get<int>(itm);
+    }
+    self->fThis->setSoundIndices(soundIndices);
+    return 0;
+}
+
+PY_GETSET_GETTER_DECL(GUIControlMod, colorScheme) {
+    PyObject* colorScheme = pyGUIColorScheme_FromGUIColorScheme(*self->fThis->getColorScheme());
+    return colorScheme;
+}
+
+PY_GETSET_SETTER_DECL(GUIControlMod, colorScheme) {
+    PY_PROPERTY_CHECK_NULL(colorScheme)
+    if (!pyPlasma_check<pfGUIColorScheme>(value)) {
+        PyErr_SetString(PyExc_TypeError, "colorScheme should be a pfGUIColorScheme");
+        return -1;
+    }
+    self->fThis->setColorScheme(((pyGUIColorScheme*)value)->fThis);
```

Comment:
`pyPlasma_check` should always be paired with `pyPlasma_get` -- see below regarding making this a pointer.

Also, need to handle `Py_None` since the colorScheme is allowed to be NULL.